nncf/experimental/torch/fx/transformations.py

```diff
@@ -669,13 +669,17 @@ def _compress_qdq_constant_transformation(model: torch.fx.GraphModule, matches)
     for match in matches:
         mul_node = match.replacements[0]
         sub_node = match.replacements[1]
-        weight_node, scale_node, zp_node, axis = None, None, None, None
         nodes_map = {node.name: match.nodes_map[node] for node in match.nodes_map}
-        get_const = partial(get_tensor_constant_from_node, model=model)
+
+        def get_const(arg: Union[torch.fx.Node, float, int]):
+            if isinstance(arg, torch.fx.Node):
+                return get_tensor_constant_from_node(arg, model)
+            return arg
+
         weight_node = get_const(nodes_map["weight"])
         scale_node = get_const(nodes_map["scale"])
         zp_node = get_const(nodes_map["zero_point"])
-        axis = nodes_map["axis"]
+        axis = nodes_map.get("axis")
```

Comment:
I think we can use the same function for axis too.
```suggestion
        axis = get_const(nodes_map["axis"])
```

Reply:
Nope, per tensor case has no "axis" key, `[]` raises key error. Intended axis value for per tensor case - None. `.get` return None https://docs.python.org/3/library/stdtypes.html#dict.get

Reply:
Sorry about that, I have updated the suggestion. I meant to say that we can pass this also to the `get_const` funciton to keep it the same as others

Reply:
But it will always be an int or None, I don't think we should check is it `torch.fx.Node` or not because it is never a `torch.fx.Node`

Reply:
Done